big5/README.md

```diff
@@ -36,8 +36,33 @@ Before using this repository, ensure you have the following installed:
 
 - [OpenSearch](https://opensearch.org) (v2.11 or later)
 - [OpenSearch Benchmark](https://opensearch.org/docs/latest/benchmark) (v1.2 or later)
+- [OpenSearch SQL Plugin](https://opensearch.org/docs/latest/search-plugins/sql/index/) (required for PPL operations)
```

Comment:
Should there be a check to ensure that the user has the SQL plug-in installed? Have you tested out what happens  and seen what error shows up if the plug-in isn't installed?